src/prebuilt/wasm2c.include.c

```diff
@@ -11,9 +11,13 @@ const char SECTION_NAME(declarations)[] =
 "\n"
 "#define TRAP(x) (wasm_rt_trap(WASM_RT_TRAP_##x), 0)\n"
 "\n"
+"#ifndef WASM_RT_NO_STACK_DEPTH_CHECKS\n"
 "#define FUNC_PROLOGUE                                            \\\n"
 "  if (++wasm_rt_call_stack_depth > WASM_RT_MAX_CALL_STACK_DEPTH) \\\n"
 "    TRAP(EXHAUSTION)\n"
+"#else\n"
+"#define FUNC_PROLOGUE\n"
+"#endif\n"
 "\n"
 "#define FUNC_EPILOGUE --wasm_rt_call_stack_depth\n"
```

Comment:
Remove here too? Probably doesn't hurt anything, but it seems odd to decrement the value here.

Reply:
Oh, not sure how I missed that, thanks...